deferred-chained-batch.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+const AbstractChainedBatch = require('abstract-leveldown').AbstractChainedBatch
+const inherits = require('inherits')
+const kOperations = Symbol('operations')
+
+function DeferredChainedBatch (db) {
```

Comment:
since we use symbols, could we also use classes? They have similar support

Reply:
Same with destructuring assignment

Reply:
For internal classes (that are not instantiated or extended externally), sure, we can use classes.

Reply:
For destructuring, do you mean:

https://github.com/Level/deferred-leveldown/blob/c197496adf54b7b44bb74838dcc342f377135ece/deferred-leveldown.js#L3

Instead:

```js
const { AbstractLevelDOWN } = require('abstract-leveldown')
const DeferredIterator = require('./deferred-iterator')
```